pkg/flare/config_check.go

```diff
@@ -99,7 +99,10 @@ func GetClusterAgentConfigCheck(w io.Writer, withDebug bool) error {
 }
 
 // PrintConfig prints a human-readable representation of a configuration
-func PrintConfig(w io.Writer, c integration.Config) {
+func PrintConfig(w io.Writer, c integration.Config, checkName string) {
+	if checkName != "" && c.Name != checkName {
```

Comment:
Note: We still show that checks are running on a host.
Potentially, that can mean that we can have an output like this
```
===== Checks on datadog-clusterchecks-5747fcc9cb-h485w =====
<nothing, as no orchestrator check runs on this host>

===== Checks on datadog-clusterchecks-5747fcc9cb-m5mwv =====

=== orchestrator check ===
Configuration provider: file
Configuration source: file:/etc/datadog-agent/conf.d/orchestrator.d/2.yaml
....